RHDH-5561: TechDocs content restructure (#877)
redhat-developer/red-hat-developers-documentation-rhdh: modules/techdocs/con-techdocs-about.adoc

:_mod-docs-content-type: CONCEPT

[id="about-techdocs_{context}"]
= About TechDocs

> Every {product} instance comes with the TechDocs plugin preinstalled and enabled by default. Your organization can use the TechDocs plugin to create, find, and use documentation in a central location and in a standardized way. You can use built-in TechDocs features and add-ons to enhance your technical documentation experience, for example:

Reviewer: ```suggestion
The {product} instance comes with the TechDocs plugin enabled as a default. You can use the TechDocs plugin to create, find, and manage documentation in a centralized and standardized manner. Also, you can use built-in TechDocs features and add-ons to enhance your technical documentation experience. For example:
```

Author: I like the change to "The {product}" and addition of "Also", but a couple things to note:

- "as a default" is incomplete, so it is more correct to write "by default" or "as a default [noun, such as _plugin_ or _setting_]". 

- _For example_ is fine, but _,for example_ is IBM style compliant.

Docs-like-code approach::
Write your technical documentation in Markdown files that are stored inside your project repository along with your code.

Documentation site generation::
Use MkDocs to create a full-featured, Markdown-based, static HTML site for your documentation that is rendered centrally in {product-short}.

Documentation site metadata and integrations::
See additional metadata about the documentation site alongside the static documentation, such as the date of the last update, the site owner, top contributors, open GitHub issues, Slack support channels, and Stack Overflow Enterprise tags.

Built-in navigation and search::

> Find the information that you want from a document more quickly and easily.

Reviewer: ```suggestion
Locate the information that you need within a document quickly and easily.
```

Author: Apply this change either immediately (and expand the scope of this PR) or with a separate CX PR.

Add-ons::

> Customize your TechDocs experience with Add-ons to address higher-order documentation needs.

Reviewer: ```suggestion
Customize your TechDocs experience with add-ons to address higher-order documentation needs.
```

Author: @hmanwani-rh the upstream docs writes _TechDocs Addon_. We added the hyphen to comply with our own standards,  but I think whether or not to capitalize should also be decided. I lean toward not capitalizing, especially since we are formatting "add-on" differently anyway, but would like to hear other stakeholder thoughts on that.
